### ISSUE TITLE
Doubt regarding merging of rgb and pose_only results

### ISSUE THREAD

**Issue by suryateja0101** [mks0601/IntegralAction_RELEASE#5]
In the merge function why only 29th file is getting considered? is it because it has more training data??

**Comment by mks0601** [comment]
what do you mean by 29th file?

**Comment by suryateja0101** [comment]
Apologies for the wrong question, I figured it out. 

1. Also i reduced the batch size to 2 for mimetics pose_only otherwise my gpu is getting memory issue. it will drop the accuracy ryt???  below is the screenshot of that.
![accuracy](https://user-images.githubusercontent.com/93408573/173180135-909eedc5-7ac1-43cb-b87c-218142e01812.jpg)

2. This is important: u have given trained data for kinetics and mimetics, but seems like it points to the same link, but how come those are generating results with different names?? did u changed the dataset name in the config file?? or am I missing something?
![doubt](https://user-images.githubusercontent.com/93408573/173180157-bfadb88b-5fa1-4ef2-a044-4818528ef5a0.png)

3. Can you also provide the dataset for NTU. In the given link only JSON files are there.

Thank you for your response.
Im going forward in this project just because you are replying :-)
